### PR TITLE
Iz add forum link GAWB-1823

### PR DESCRIPTION
**Pull request by zarsky-broad**
Requires adding key to configs:
https://github.com/broadinstitute/firecloud-develop/pull/401
https://github.com/broadinstitute/firecloud-develop/pull/402
https://github.com/broadinstitute/firecloud-develop/pull/403

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by MattPutnam — APPROVED** [review]
Code looks good.  I don't like how the name dropdown looks like a combo box but the items don't span the full width of it.  I defer to you on that though.

File: src/cljs/main/broadfcui/common.cljs [hunk 1]
```diff
@@ -260,27 +260,27 @@
...
+     (let [{:keys [contents dropdown-class]} props
            {:keys [dropdown-container dropdown-id]} @locals]
        (react/render
         (react/create-element
          ;; empty string makes react attach a property with no value
-         [:div {:className "dropdown-pane" :id dropdown-id :data-dropdown ""
+         [:div {:className (str "dropdown-pane " dropdown-class) :id dropdown-id :data-dropdown ""
                 :ref (this :-create-dropdown-ref-handler)
-                :style {:whiteSpace "normal"}}
+                :style (merge {:whiteSpace "normal"} (:style-override props))}
```
Comment (dmohs) [inline src/cljs/main/broadfcui/common.cljs]: By convention, we usually name this key `:style`.

**Comment by zarsky-broad** [comment]
@MattPutnam Ask and ye shall receive, good note

File: src/cljs/main/broadfcui/common.cljs [hunk 2]
```diff
@@ -260,27 +260,31 @@
...
        (react/render
         (react/create-element
          ;; empty string makes react attach a property with no value
-         [:div {:className "dropdown-pane" :id dropdown-id :data-dropdown ""
+         [:div {:className (str "dropdown-pane " dropdown-class) :id dropdown-id :data-dropdown ""
                 :ref (this :-create-dropdown-ref-handler)
-                :style {:whiteSpace "normal"}}
+                :style (merge
+                        {:whiteSpace "normal"}
+                        (:style-override props)
```
Comment (dmohs) [inline src/cljs/main/broadfcui/common.cljs]: Keep the `:style` convention we use elsewhere unless you can argue this is better and we should change it everywhere.

**Review by dmohs — APPROVED** [review]
Looks good to me. I have an idea about a convention for generic component style overrides that I'd like to bounce off of you.

**Pull request merged**